charts/values.yaml

```diff
@@ -33,5 +33,6 @@ operator:
       defectDojoVerified: "false"
     image:
       repository: ghcr.io/telekom-mms/docker-trivy-dojo-operator
-      tag: trivy-dojo-report-operator-0.4.1@sha256:0e3111cb58256ee2ef8a9320f1c4d1d3097ae63a76169b2cbc3d45ca93b3ca17
+      tag: myvalue
```

Comment:
Why do you change this?

Reply:
That was an error on my side, will revert